python/smqtk/codebook/__init__.py

```diff
@@ -0,0 +1,4 @@
+# TODO -- is this necessary?
+
+class Codebook:
```

Comment:
If you have a work-in-progress class that's not implemented and not used anywhere, don't include it in a commit.